Dockerfile

```diff
@@ -6,6 +6,12 @@ RUN ls .
 ADD . /gatk
 WORKDIR /gatk
 
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```

Comment:
`git-lfs` is in the official package repositories for both ubuntu 18.04 and ubuntu 20.04 -- are you sure that you need this step?